.travis.yml

```diff
@@ -5,25 +5,52 @@ cache:
     - $HOME/.composer/cache
 
 php:
-    - 5.5
-    - 5.6
-    - 7.0
+    - "5.5"
+    - "5.6"
+    - "7.0"
+    - "7.1"
+    - nightly
     - hhvm
 
 matrix:
     fast_finish: true
     include:
-        - php: 7.0
-          env: SYMFONY_VERSION=2.8.*
-        - php: 7.0
-          env: SYMFONY_VERSION=3.0.*
-        - php: 7.0
+        - php: "5.5"
+          env:
+              - "SYMFONY_VERSION=~2.8.0"
+              - "COMPOSER_FLAGS=--prefer-lowest"
+        - php: "7.0"
+          env: "SYMFONY_VERSION=~3.0.0"
+        - php: "7.0"
+          env: "SYMFONY_VERSION=~3.1.0"
+        - php: "7.0"
+          env: "SYMFONY_VERSION=~3.2.0@dev"
+    allow_failures:
+        - php: nightly
```

Comment:
Having it in allow_failure looks like a good compromise. We can check manually but it doesn't mark the build in red because of a non-stable version of PHP.